src/openrct2/interface/Chat.h

```diff
@@ -14,10 +14,10 @@
 
 #include <string_view>
 
-#define CHAT_HISTORY_SIZE 10
-#define CHAT_INPUT_SIZE 1024
-#define CHAT_MAX_MESSAGE_LENGTH 200
-#define CHAT_MAX_WINDOW_WIDTH 600
+constexpr int8_t kChatHistorySize = 10;
+constexpr int16_t kChatInputSize = 1024;
+constexpr int kChatMaxMessageLength = 200;
```

Comment:
Please use one of the explicitly sized integers instead.

Reply:
Sorry, it was giving errors due to the std::min function - I have casted it instead...